cron/centAcl.php

```diff
@@ -110,9 +111,29 @@
         programExit($errorMessage);
     }
 
-    $resourceCache = array();
+    /**
+     * Sync ACL with LDAP's contactgroup
+     * If the LDAP is enable and the last check is greater than the update period
+     *
+     * @TODO : Synchronize LDAP with contacts data in background to avoid it at login
+     */
+    $queryOptions = "SELECT `key`, `value` FROM `options` WHERE `key` IN ('ldap_auth_enable', 'ldap_last_acl_update')";
+    $res = $pearDB->query($queryOptions);
+    while ($row = $res->fetch()) {
+        switch ($row['key']) {
+            case 'ldap_auth_enable':
+                $ldap_enable = $row['value'];
+                break;
+            case 'ldap_last_acl_update':
+                $ldap_last_update = $row['value'];
+                break;
+        }
+    }
+    if ($ldap_enable == 1 && $ldap_last_update < (time() - LDAP_UPDATE_PERIOD)) {
```

Comment:
$ldap_enable and $ldap_last_update are not initalized if no one ldap is configured
Furthermore, you should use camelcase for variable name

Reply:
Same comments as @kduret